packages/manager/apps/pci-billing/src/api/hook/useConsumption.tsx

```diff
@@ -11,8 +11,14 @@ import {
 const roundPrice = (num: number, fractionDigits = 2) =>
   Number(num.toFixed(fractionDigits));
 
-const sumPrices = <T,>(items: T[], key: string) =>
-  roundPrice(items?.reduce((sum, item) => sum + item[key], 0));
+const sumPrices = (
+  items: {
+    region: string;
+    quantity: TQuantity;
+    name: string;
+    totalPrice: number;
+  }[],
```

Comment:
May you please declare the type explicitly, so we can reuse it to type function parameters?